src/ssb_project_cli/ssb_project/app.py

```diff
@@ -806,28 +805,32 @@ def create_github(
     Returns:
         str: Repository url
     """
-    private_repo = True if repo_privacy != "public" else False
-
     g = Github(github_token)
+
     if not debug_without_create_repo:
-        try:
-            g.get_organization(GITHUB_ORG_NAME).create_repo(
-                repo_name,
-                private=private_repo,
-                auto_init=False,
-                description=repo_description,
-            )
-        except BadCredentialsException:
+        url = f"https://api.github.com/orgs/{GITHUB_ORG_NAME}/repos"
```

Comment:
Nice! Could you just write a comment on why we're not using the `create_repo` method from the `Github` package  (i.e. that it doesn't support visibility)?

Reply:
I mean, just a one-liner in the code. This PR is well documented